documentation/book/proc-deploying-cluster-operator-kubernetes-to-watch-multiple-namespaces.adoc

```diff
@@ -9,10 +9,19 @@
 
 * Edit the installation files according to the {Namespace} the Cluster Operator is going to be installed in.
 +
+On Linux, use:
++
 [source, subs="+quotes"]
 ----
 sed -i 's/namespace: .\*/namespace: _my-namespace_/' examples/install/cluster-operator/*RoleBinding*.yaml
 ----
++
+On MacOS, use:
```

Comment:
Do we want to wrap these in `ifdef`, for downstream?

Reply:
Why? I think this should be downstream as well.